projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,222 @@
+                  {{ 'orderHistory.startOn' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.firstDate | cxDate }}</a
+                >
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.frequency' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.displayTimeTable }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.nextOrderDate' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.activationTime | cxDate }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.total' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.subTotal.formattedValue }}</a
+                >
+              </td>
+              <td (click)="openDialog()">
+                <a
+                  (click)="openDialog()"
+                  class="cx-order-cancel btn btn-link"
+                  #element
+                >
+                  {{ 'orderHistory.cancel' | cxTranslate | uppercase }}
```

Comment:
actually instead of putting it uppercase here from what I said before when you made all uppercase in the i18 object, let's put it in the stylesheet instead

however, you are missing the underline text for `cancel` as how this new thing looks like in the wires.

Reply:
button needs needs to be aligned right as well from the wires on mobile